musicbrainz/csv/get_genre.py

```diff
@@ -0,0 +1,62 @@
+"""
+Script for retrieving the "genre" dumps since they are not available online.
+"""
+
+import time
+import requests
+from requests.exceptions import HTTPError
+import pandas as pd
+from bs4 import BeautifulSoup
+
+URL = "https://musicbrainz.org/ws/2/genre/all"
+PARAMS = dict(fmt="json", limit="50")
+
+data = []
+max_records = dict(requests.get(url=URL, params=PARAMS, timeout=500).json())[
```

Comment:
Do you need dict here? The json method will return a python data structure (dict or list) depending on the json object returned.